templates/repo/scripts/common.mk

```
@@ -6,6 +6,7 @@ TF_VARS := $(patsubst %,-e%,$(filter TF_VAR_%,$(.VARIABLES)))
 REPO_ROOT := $(shell git rev-parse --show-toplevel)
 REPO_RELATIVE_PATH := $(shell git rev-parse --show-prefix)
 AUTO_APPROVE := false
+AWS_SDK_LOAD_CONFIG := 1
```

```suggestion
export AWS_SDK_LOAD_CONFIG := 1
```
I think we have to do that, but I'm not 100% sure